tests/php/ORM/DBFieldTest.php

```diff
@@ -424,12 +425,14 @@ public function testSaveIntoRespectsSetters()
 
     public function testDefaultValues(): void
     {
+        $dataObjectClasses = ClassInfo::subclassesFor(DataObject::class, false);
         $expectedBaseDefault = null;
         $expectedDefaults = [
             DBBoolean::class => false,
             DBDecimal::class => 0.0,
             DBInt::class => 0,
             DBFloat::class => 0.0,
+            DBClassName::class => $dataObjectClasses[array_key_first($dataObjectClasses)],
```

Comment:
Note that the default value passed into the database was _always_ this value (see https://github.com/silverstripe/silverstripe-framework/issues/11165). This test was just checking the result of `getDefaultValue()` but the value being passed to the database came from `DBClassName::getDefault()`.

So now `getDefaultValue()` will give you _the actual default value_ which is correct, so updating this test is just now reflecting the accurate value.